.circleci/config.yml

```diff
@@ -56,146 +74,72 @@ commands:
-          command: |
-            sudo apt-get install -y libpng-dev sqlite3
-            sudo docker-php-ext-install gd
-
-executors:
-  node-executor:
-    docker:
-      - image: circleci/node:10-browsers
-  php-node-executor:
-    docker:
-      - image: circleci/php:7.1-fpm-node-browsers
-
 jobs:
-  lint_unit:
-    executor:
-      name: node-executor
+  test_node:
+    machine: true
     steps:
-    - build
-    - save_cache:
-        key: *node-modules-key
-        paths:
-        - node_modules
-    - run: yarn workspace admin-ui test:lint
-    - run: yarn workspace admin-ui test:unit
-    - store_test_results:
-        path: reports
+      - start-project
+      - run: docker exec -it drupal_admin_ui_drupal drush user:password admin admin
+      - run: docker exec -it -e REACT_APP_DRUPAL_BASE_URL=http://drupal drupal_admin_ui_node yarn workspace admin-ui build
+      - run: docker exec -it -e REACT_APP_DRUPAL_BASE_URL=http://drupal -e NIGHTWATCH_LOGIN_admin_PASSWORD=admin drupal_admin_ui_node yarn test
+      - store_test_results:
+          path: ~/project/packages/admin-ui/reports
 
-  integration_phpunit_nightwatch:
-      executor:
-        name: php-node-executor
-      steps:
-        - fix-php-env
-        - build
-        - restore_cache:
-            keys:
-            - *composer-cache-key
-        - run:
-            name: Install Drupal
-            command: |
-                cd ~/project/demo
-                composer config repositories.repo-name path "~/project/admin_ui_support"
-                composer require justafish/drupal-admin-ui-support:dev-$CIRCLE_BRANCH
-                composer install
-                composer setup
-        - save_cache:
-            key: *composer-cache-key
-            paths:
-            - ~/.composer/cache
-        - run:
-            name: Start Drupal and run Nightwatch and PHPUnit tests
-            environment:
-              REACT_APP_DRUPAL_ROOT: ~/project/demo
-              REACT_APP_DRUPAL_BASE_URL: http://127.0.0.1:8888
-              SIMPLETEST_BASE_URL: http://localhost:8888
-              SYMFONY_DEPRECATIONS_HELPER: disabled
-            command: |
-              cd ~/project/demo
-              composer start &
-              sleep 10
-              cd ~/project
-              yarn workspace admin-ui start &
-              sleep 20
-              yarn workspace admin-ui test:functional
-              cd ~/project/demo
-              ./vendor/bin/phpunit -c docroot/core --log-junit reports/phpunit.xml docroot/modules/contrib/drupal-admin-ui-support
-        - store_test_results:
-            path: ~/project/packages/admin-ui/reports/nightwatch
-        - store_artifacts:
-            path: ~/project/packages/admin-ui/reports/nightwatch
-        - store_test_results:
-            path: ~/project/reports
+  test_drupal:
+    machine: true
+    steps:
+      - start-project
+      - run: docker exec -it drupal_admin_ui_drupal composer test
```

Comment:
We can try `docker exec -it drupal_admin_ui_drupal su -m apache -s /bin/bash -c "composer test"`